macrosAkka/src/main/scala/com/softwaremill/macwire/macwireakka/MacwireAkkaMacros.scala

```diff
@@ -0,0 +1,14 @@
+package com.softwaremill.macwire
+package macwireakka
```

Comment:
Ah, didn't think about that, good catch :)

I don't have any good ideas about that really ... maybe `akkaActors`? But then it's the double-a ;)